spec/ruby/core/integer/coerce_spec.rb

```diff
@@ -1,104 +1,107 @@
 require_relative '../../spec_helper'
 
-require 'bigdecimal'
-
-describe "Integer#coerce" do
-  context "fixnum" do
-    describe "when given a Fixnum" do
-      it "returns an array containing two Fixnums" do
-        1.coerce(2).should == [2, 1]
-        1.coerce(2).map { |i| i.class }.should == [Integer, Integer]
-      end
-    end
+ruby_version_is ""..."3.4" do
```

Comment:
This disables all Integer#coerce specs, it's not OK.
Same for Rational#coerce specs below.